layouts/partials/hooks/head-end.html

```diff
@@ -25,3 +25,23 @@
   <link rel="canonical" href="{{ $canonicalURL }}" data-proofer-ignore>
 
 {{- end -}}
+
+<!-- Kapa.ai Chat Widget -->
+<script 
+  async 
+  src="https://widget.kapa.ai/kapa-widget.bundle.js" 
+  data-website-id="2b645d39-e001-4c2b-bfbd-aa4f25453385"
+  data-project-name="OpenTelemetry"
+  data-project-color="#f5a800"
+  data-project-logo="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png"
+  data-button-hide="true"
+  data-modal-override-open-id="kapa-ai-trigger"
+  data-modal-border-radius="8px"
+  data-modal-header-bg-color="#4f62ad"
+  data-modal-title="OpenTelemetry Docs AI"
+  data-modal-title-color="#ffffff"
+  data-modal-body-bg-color="#ffffff"
+  data-query-input-border-color="#f5a800"
+  data-modal-disclaimer="This is an experimental feature that uses AI to answer questions using the documentation, OpenTelemetry GitHub, StackOverflow, and other resources. Please give us feedback in [this GitHub issue](https://github.com/open-telemetry/opentelemetry.io/issues/6658)."
```

Comment:
```suggestion
  data-modal-disclaimer="This is an experimental feature that uses AI to answer questions based on this site's documentation, OpenTelemetry GitHub, StackOverflow, and other resources. Please give us feedback in [this GitHub issue](https://github.com/open-telemetry/opentelemetry.io/issues/6658)."
```

Reply:
Also, we might want folks to open new issues rather than comment of the existing one?

Reply:
I was thinking for now it'd be good to try and consolidate the feedback in one place.

Reply:
I am fine with both, as long as we give people the option to provide feedback